### PR TITLE
rsc: Include migration tool in wake release

### PR DESCRIPTION
**Pull request by V-FEXrt**

### CODE REVIEW

File: build.wake [hunk 1]
```diff
@@ -107,12 +107,17 @@ def doInstall dest kind =
     # kick off in parallel, but rsc and rsc_tool cannot be built at the same time right now.
     # Force them to be serial relative to each other.
     def install_rsc_bins =
+        require Pass rsc_migration = buildRSCMigration Unit
+
+        require Pass rsc_migration =
+            installAs "{dest}/bin/wake-remote-cache-migration" rsc_migration
```
Comment (V-FEXrt) [inline build.wake:113]: Tempted to name these `wake-rsc`, `wake-rsc-tool`, and  `wake-rsc-migration` instead if anyone has any opinions

Comment (colinschmidt) [reply build.wake:113]: I think using `rsc` in the tool names is a good idea.

File: build.wake [hunk 2]
```diff
@@ -107,12 +107,17 @@ def doInstall dest kind =
     # kick off in parallel, but rsc and rsc_tool cannot be built at the same time right now.
     # Force them to be serial relative to each other.
     def install_rsc_bins =
+        require Pass rsc_migration = buildRSCMigration Unit
```
Comment (colinschmidt) [inline build.wake:110]: I think the comment above should be updated to mention that RSCMigration is also part of the serial chain, or if thats not a constraint move the require on RSCMigraiton later.

**Review by colinschmidt — APPROVED** [review]
LGTM

**Pull request merged**